### ISSUE TITLE
MoL loss for other models

### ISSUE THREAD

**Issue by everRoc** [princeton-vl/SEA-RAFT#3]
Great work! I wonder whether MoL loss is helpful for other methods, but I don't find the ablations in the paper. Have you ever directly adopted MoL loss on other models like RAFT and GMA?

**Comment by MemorySlices** [comment]
Thanks for the interest! Our work is very similar to RAFT and we have provided ablations. Unfortunately, we did not adopt MoL on other models such as GMA. According to our experience, the training settings of MoL and L1 for the same model are almost the same :D. You can have a quick ablation by replacing L1 with MoL according to the details mentioned in the paper.

**Comment by everRoc** [comment]
您好，您的邮件我已经收到了，谢谢
